contracts/src/v0.8/ccip/offRamp/MessageTransformerOffRamp.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.24;
+
+import {IMessageTransformer} from "../interfaces/IMessageTransformer.sol";
+import {Internal} from "../libraries/Internal.sol";
+import {OffRamp} from "./OffRamp.sol";
+
+/// @notice OffRamp that uses a message transformer to transform messages before execution
+contract MessageTransformerOffRamp is OffRamp {
```

Comment:
Can we name them OffRampWithMsgTransformer instead? They're primarily a ramp so the name should reflect that imo